src/patternfly/patternfly-date-picker.scss

```diff
@@ -0,0 +1,177 @@
+// stylelint-disable
+.pf-c-date-picker__calendar {
+  &.flatpickr-calendar {
+    // calendar
+    --pf-c-date-picker__flatpickr__calendar--Background: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__calendar--Padding: var(--pf-global--spacer--md);
+    --pf-c-date-picker__flatpickr__calendar--open--ZIndex: var(--pf-global--ZIndex--sm);
+
+    // month/year section
+    --pf-c-date-picker__flatpickr__months--PaddingBottom: var(--pf-global--spacer--md);
+    --pf-c-date-picker__flatpickr__month--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+    --pf-c-date-picker__flatpickr__month--PaddingTop: 10px;
```

Comment:
can you ask design if theyd be ok with using 8px to fit in with the system ?

Reply:
This makes the month/year area align with the arrows beside it. It's not creating a visible 10px gap from a border (which wouldn't align with our system), so I don't think there is much value in using our spacer vars. This is what it looks like with 8px.

<img width="308" alt="Screen Shot 2020-09-22 at 5 53 28 PM" src="https://user-images.githubusercontent.com/35148959/93945597-e01a4e80-fcfc-11ea-904d-e78b3cd0a5dc.png">

and here it is with 10

<img width="340" alt="Screen Shot 2020-09-22 at 5 53 35 PM" src="https://user-images.githubusercontent.com/35148959/93945614-e6a8c600-fcfc-11ea-8023-281e345ad05d.png">

I think we should wait until design can look at it and then we can update the spacing if it doesn't look right.